Data_Structures_Answers.md

```diff
@@ -1,18 +1,25 @@
-Add your answers to the questions below.
+ Add your answers to the questions below.
 
 1. What is the runtime complexity of your ring buffer's `append` method?
+O(1)
 
 2. What is the space complexity of your ring buffer's `append` function?
+O(1)
 
 3. What is the runtime complexity of your ring buffer's `get` method?
+O(n)
 
 4. What is the space complexity of your ring buffer's `get` method?
-
+O(2n) -> O(n)
 
 5. What is the runtime complexity of the provided code in `names.py`?
+O(n^2)
 
 6. What is the space complexity of the provided code in `names.py`?
+O(2n) -> O(n)
```

Comment:
Hey Chance! I don't know what was talked about during your 1-1 but the space complexity for the provided code is O(1).